privacy_partner_report/views/privacy_report.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+    <record id="action_report_partner_data_xlsx" model="ir.actions.report.xml">
+      <field name="name">Privacy Partner Data to Excel</field>
```

Comment:
I'd remove "to Excel" from here. The file suffix is explicit enough.

Reply:
Yes. It's because before there was also an option to pdf.